### PR TITLE
Feature/release related issues

### PR DESCRIPTION
**Pull request by eskilsen**
Another update before release. Some bugs was found.
## Description

- Update loading appearance on buttons. Make loading be possible for all buttons in component preview.
- Update logos in logotype section
- Accessibility improvements
- Minor updates on sheet component

## How Has This Been Tested?

Ran all existing tests and update snapshots.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Review by eskilsen — COMMENTED** [review]
Nice! :D

File: src/App/components/Sheet/index.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ const Sheet = ({ id, requireAction, title, footer, children }) => (
             </div>
             {footer &&
                 <div className="sheet-footer">{"\n"}
```
Comment (eskilsen) [inline src/App/components/Sheet/index.js:19]: add .button-group class and remove ml-1 and mr-1? 🤔

**Review by eskilsen — COMMENTED** [review]
👍 nice man!

**Review by PhungNg — APPROVED** [review]
Leggo! 🏃

**Pull request merged**